simple_parsing/wrappers/field_wrapper.py

```diff
@@ -95,11 +94,11 @@ def arg_options(self) -> Dict[str, Any]:
         return self._arg_options
 
     def __call__(
-        self,
-        parser: argparse.ArgumentParser,
-        namespace: argparse.Namespace,
-        values: Any,
-        option_string: Optional[str] = None,
+            self,
+            parser: argparse.ArgumentParser,
+            namespace: argparse.Namespace,
+            values: Any,
+            option_string: Optional[str] = None,
```

Comment:
Your formatter does it wrong. Check for other cases too please

Reply:
Please commit a config file for black. I just used pycharm's pep8 based formatting.

Reply:
Tha's a good idea! Don't worry about formatting for now though, I'll take care of that.

Reply:
I formatted with black's defaults, not too many things have changed so I guess it's mostly the same config.